### PR TITLE
PR for SRVCOM-2388: Updated the procedure for "Deleting Operators from a cluster using the CLI" section.

### PR DESCRIPTION
**Pull request by kaldesai**
**Affected versions for cherry-picking:** **4.11+** I have made the changes in the Openshift-main branch. (Changes are related to Serverless) 

**Tracking JIRA:** https://issues.redhat.com/browse/SRVCOM-2388

**Doc preview:** [Deleting Operators from a cluster using the CLI](https://66454--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-deleting-operators-from-cluster#olm-deleting-operator-from-a-cluster-using-cli_olm-deleting-operators-from-a-cluster)

**Description:** Updated the procedure for **Deleting Operators from a cluster using the CLI** section and replaced `jaeger` with `serverless-operator`.

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM from QE

File: modules/olm-deleting-operators-from-a-cluster-using-cli.adoc [hunk 1]
```diff
@@ -22,41 +22,41 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. Check the current version of the subscribed Operator (for example, `jaeger`) in the `currentCSV` field:
+. Check the current version of the subscribed Operator (for example, `serverless-operator`) in the `currentCSV` field:
```
Comment (nalhadef) [inline modules/olm-deleting-operators-from-a-cluster-using-cli.adoc:25]: What is the Operator being checked for?

If the user is just checking that the subscribed Operator is defined, I'd reword the sentence to: "Ensure the latest version of the subscribed operator is identified in the **currentCSV** field."

Comment (kaldesai) [reply modules/olm-deleting-operators-from-a-cluster-using-cli.adoc]: @nalhadef This is a wonderful suggestion. Thank you so much! I have incorporated it.

**Comment by kaldesai** [comment]
@nalhadef Can you please ack the changes and add the `peer-review-done` label?

File: modules/olm-deleting-operators-from-a-cluster-using-cli.adoc (hunk 1 above)
Comment (nalhadef) [reply modules/olm-deleting-operators-from-a-cluster-using-cli.adoc]: Thanks, @kaldesai!

**Comment by nalhadef** [comment]
/lgtm

Ack.

**Comment by nalhadef** [comment]
/remove-label peer-review-in-progress
label peer-review-done

**Comment by nalhadef** [comment]
Done. Thanks, Kalyani.

**Review by bscott-rh — APPROVED** [review]
